gtwrap/pybind_wrapper.py

```diff
@@ -126,6 +126,7 @@ def _wrap_method(self, method, cpp_class, prefix, suffix, method_suffix=""):
 
         if method.name == 'print':
             type_list = method.args.to_cpp(self.use_boost)
+            ret = ret.replace('self->', 'py::scoped_ostream_redirect output; self->')
```

Comment:
Awesome but add the Pybind link to this as a reference explaining the rationale?

Reply:
done!